relay-filter/src/csp.rs

```diff
@@ -38,8 +49,11 @@ pub fn should_filter(event: &Event, config: &CspFilterConfig) -> Result<(), Filt
 /// or None (matches anything in the respective position)
 #[derive(Hash, PartialEq, Eq)]
 pub struct SchemeDomainPort {
+    /// the scheme of the url
```

Comment:
Same for the following doc comments.

```suggestion
    /// The scheme of the url.
```

Reply:
yep